components/range_cache_memory_engine/src/background.rs

```diff
@@ -956,34 +970,90 @@ impl Runnable for BackgroundRunner {
                                     }
                                     Err(e) => {
                                         error!("creating rocksdb iterator failed"; "cf" => cf, "err" => %e);
-                                        return false;
+                                        return None;
                                     }
                                 }
                             }
-                            true
+
+                            // gc the range
+                            let tso_timeout = std::cmp::min(gc_interval, Duration::from_secs(5));
```

Comment:
Please add more comments / annotations on the default value `5 seconds` and make it as a constant value.

Reply:
Done